index.html

```diff
@@ -1,37 +1,200 @@
+        <div class="project_info">
+          <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+          <ul class="languages_used">
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">html</a></li>
+          </ul>
+          <a class="see_project" href="#">See Project</a>
+        </div>
       </div>
+    </div>
+  </section>
+  <section class="about_me">
+    <div class="detailed_info">
+      <h2 class="title">About me</h2>
+      <p class="aboutme">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <a class="see_project" href="#">Get My Resume</a>
+    </div>
+    <ul class="card2">
+      <li class="card2_content">
+        <img src="./images/icon-languages.svg" alt="icon-languages" />
+        <h4>Languages</h4>
+        <ul class="languages">
+          <li><a href="#">JavaScript</a></li>
+          <li><a href="#">Ruby</a></li>
+          <li><a href="#">Html</a></li>
+          <li><a href="#">CSS</a></li>
+        </ul>
+      </li>
+      <li class="card2_content">
+        <img src="./images/framework__icon.svg" alt="framework__icon" />
+        <h4>Frameworks</h4>
+        <ul class="languages">
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby on rails</a></li>
+          <li><a href="#">RSpec</a></li>
+          <li><a href="#">Capybara</a></li>
+          <li><a href="#">Selenium</a></li>
+        </ul>
+      </li>
+      <li class="card2_content">
+        <img src="./images/skills_icon.svg" alt="skills_icon" />
+        <h4>Skills</h4>
+        <ul class="languages">
+          <li><a href="#">Codekit</a></li>
+          <li><a href="#">GitHub</a></li>
+          <li><a href="#">Codepen</a></li>
+          <li><a href="#">Gitlab</a></li>
+          <li><a href="#">Terminal</a></li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+  <footer class="contact_form">
+    <p class="send_me_msg">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+      touch.</p>
+    <form action="https://formspree.io/f/xzbopjvn" method="POST">
+      <ul class="form_items">
+        <li><input class="box_Style" type="text" name="name" id="name" maxlength="30" placeholder="Full Name" required>
+        </li>
+        <li><input class="box_Style" type="email" name="email" id="email" placeholder="Email address" required></li>
+        <li><textarea name="message" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500"
+            required></textarea></li>
+        <li><input type="button" id="get_in_touch" class="see_project" value="Get in touch"></li>
+      </ul>
+    </form>
```

Comment:
- [ ] [OPTIONAL]: I want to start by pointing out that having a list inside a form is not incorrect. But in order to keep your code clean, you should consider removing them, if you want to keep their style, you should consider adding the form_items class inside the `<form>`.
- [ ] Right now, whenever I click on the submit button, nothing happens. I didn't get any validation messages (I tried clicking on the button with all inputs blank), this might be happening because you are using the `type="button"`, you should consider changing it to `type="submit"` instead. You can read more about buttons type [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
> Keep in mind that according to the Formspree documentation, the button type should be "submit", as you can see here:

![image](https://user-images.githubusercontent.com/66967274/144059586-a139bb30-2d0f-4e2c-a75b-4c06d9ae8d65.png)